modules/common/react/lib/theming/index.ts

```diff
@@ -1,4 +1,8 @@
+import CanvasProvider from './CanvasProvider';
+
 export * from './breakpoints';
 export * from './createCanvasTheme';
 export * from './types';
+export * from './styled';
 export * from './theme';
+export {CanvasProvider};
```

Comment:
Can you change `CanvasProvider` to a named export so it's consistent with the other packages here?